src/routes/docs/sdks/+page.markdoc

```diff
@@ -103,7 +103,7 @@ Server libraries for integrating with Appwrite to build server side integrations
 ---
 * {% only_dark %}{% icon_image src="/images/platforms/dark/go.svg" alt="Go logo" size="m" /%}{% /only_dark %}
 {% only_light %}{% icon_image src="/images/platforms/go.svg" alt="Go logo" size="m" /%}{% /only_light %}
-* Go SDK `1.22`
+* Go SDK `1.23`
```

Comment:
This should be the release version of the SDK

Reply:
reverted